tools/pipelines/build-api-markdown-documenter.yml

```diff
@@ -34,12 +34,12 @@ trigger:
     - .prettierignore
     - tools/api-markdown-documenter
     - tools/pipelines/build-api-markdown-documenter.yml
-    - tools/pipelines/templates/build-npm-package.yml
+    - tools/pipelines/templates/1ES/build-npm-package.yml
     - tools/pipelines/templates/include-set-package-version.yml
-    - tools/pipelines/templates/include-vars.yml
+    - tools/pipelines/templates/1ES/include-vars.yml
```

Comment:
Tied to a comment above, if we undo the change to include-vars we should revert this to use the non-1ES path, same for the other trigger below (line 62), and for the actual use of the template (line 69).

Reply:
Updated.